modules/yardstick/DEVNOTES.txt

```diff
@@ -0,0 +1,34 @@
+Building from Ignite sources
+=========================================
+
+1. Create a local clone of Ignite repository
+
+https://git-wip-us.apache.org/repos/asf/ignite
+
+2. Run
+
+mvn clean package -Pyardstick -pl modules/yardstick -am -DskipTests
+
+in the Apache Ignite root directory.
+
+This command will compile the project and also unpack scripts from `yardstick-resources.zip` file to
+`modules/yardstick/target/assembly/bin` directory.
+
+Artifacts can be found in `modules/yardstick/target/assembly` directory.
+
+
+Building from standalone sources
+================================
+
+Run following maven command:
```

Comment:
"Building from standalone sources" is the building instruction for Ignite Yardstick source files included in Ignite binary release and "Building from Ignite Sources" is the building instruction for building Yardstick from modules/yardstick in Ignite source files.